### ISSUE TITLE
"print sign error" in gotoPrint()

### ISSUE THREAD

**Issue by LouisdeBruijn** [minvws/nl-covid19-coronacheck-website#38]
After asking for the recovery certificate and going through the authentication with DigiD to retrieve positive test results, the last step (big blue button "Get QR code") catches an error 'print sign error'. 

The full message is 'Contact our helpdesk and give them an error code (print sign error) or try again later.'

I went through the codebase and had a look, it seems to be [here](https://github.com/minvws/nl-covid19-coronacheck-website/blob/bbdd6fbae3f265f2a792fb989859ab9b9eaf077e/print-portal/src/components/views/3-collect/_shared/overview-mixin.vue#L62). I wonder if the remarks on L65&L66 are already resolved since it's July 1st and the check should now also be done for EU. 

I might be out on a limb here, but I'd thought I'd contribute something. Thanks!

PS: issue perseveres on both NL and EN

**Comment by anneke** [comment]
Thanks Louis! We appreciate you taking the time to make us aware of this error. I've passed it on to my colleagues to find out more!

**Comment by jeroenbreen** [comment]
Hi Louis, supplementary to Anneke's comment:
you are right that the comment is outdated (will remove). But currently we check for both as you can see at
https://github.com/minvws/nl-covid19-coronacheck-website/blob/bbdd6fbae3f265f2a792fb989859ab9b9eaf077e/print-portal/src/components/views/3-collect/_shared/overview-mixin.vue#L68

But the reason that your signing is rejected can not be explained from the frontend. 

If you can get information to us, about what is in your 'events'. We could try to find out what the reason is for backend saying NO. Please do this via the helpdesk, I believe they are quite responsive. Please be aware that there might be privacy sensitive information inside this 'payload'. 
You can also look for yourself: just do `JSON.parse(atob(payload))` and you will see all the properties of your 'event' (the recovery). My first guess is: essential properties are missing (in that case GGD/RIVM would have to fix).

![Screenshot 2021-07-02 at 18 36 37](https://user-images.githubusercontent.com/6692653/124304744-715c0280-db64-11eb-8748-c0978a08b5bd.png)

**Comment by jeroenbreen** [comment]
Btw to be sure. Did you had a 500 on the `print` endpoint?

**Comment by anneke** [comment]
@LouisdeBruijn Could you take another look? :)

**Comment by LouisdeBruijn** [comment]
﻿Hi all,

Thanks for the response. I was superinvested because I was going to go on holidays and subsequentpy missed this thread. I just returned this weekend!

So I did a functional test again and it seems to be working again. I didn’t get the error again. I didn’t check if it was a 500 code when I got it two weeks ago, sorry.

It seems to be resolved, thanks for the emails! 

Greetings,

Louis de Bruijn
+31 6 15443390
